### PR TITLE
add feature fuzzy matching

### PR DESCRIPTION
**Pull request by zhangweijie-cn**
R= @yorkie

### CODE REVIEW

**Comment by zhangweijie-cn** [comment]
@yorkie

File: lib/utils.js [hunk 1]
```diff
@@ -1,14 +1,28 @@
 'use strict';
 
-function getMasksFromCommentBody(commentBody) {
+const FuzzyMatching = require('fuzzy-matching');
```
Comment (yorkie) [inline lib/utils.js]: Do you think about using https://github.com/sindresorhus/leven or https://github.com/NaturalNode/natural to instead?

**Comment by zhangweijie-cn** [comment]
@yorkie

**Comment by yorkie** [comment]
LGTM and @seanbot bump patch

**Pull request merged**